metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -562,6 +562,18 @@ case class DebugUnresolvedMainClassParams(
     @Nullable envFile: String = null
 )
 
+final case class DebugScalaTestSelectionParams(
+    @Nullable target: b.BuildTargetIdentifier,
+    @Nullable classes: java.util.List[ScalaTestSuiteSelection],
```

Comment:
Can we even run mutiple test classes currently?

Reply:
Do we need to make it universal? Multiple test cases can be run separately in different sessions, which is probably a better and more scalable approach.

Reply:
Each session has like ~1s overhead so I'm not a fan of multiplying it by N. 

>Do we need to make it universal?

What's wrong with being universal, but simple and concise at the same time? Allowing multiple suites doesn't make this much harder, nor does allow only a single suite make this much easier. And it's compliant with already existing implementations of `DebugSessionParamsDataKind.SCALA_TEST_SUITES`